src/components/Modal/GPOSWizard/Vote/Vote.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import Translate from 'react-translate-component';
+import Voting from '../../../Voting/VotingContainer';
+
+
+class Vote extends React.Component {
+  render() {
+    return(
+      <div className='gpos-modal__voting'>
+        <div className='gpos-modal__voting--left'>
+          <Translate
+            component='p'
+            className='txt'
+            content='gpos.wizard.vote'
+          />
+        </div>
+        <div className='gpos-modal__voting--right'>
+          <Voting handlers={ {cancel: this.props.cancelHandler, finish: this.props.finishHandler} }/>
+        </div>
+      </div>
+    );
+  }
+}
+
+/* eslint-disable-next-line */
```

Comment:
why you have this? Going to implement later?

Reply:
uhh idk haha part of boilerplate redux snippet I have. Will remove